UnitsNet/UnitDefinitions/Pressure.json

```diff
@@ -107,17 +107,17 @@
       "PluralName": "NewtonsPerSquareMeter",
       "FromUnitToBaseFunc": "x",
       "FromBaseToUnitFunc": "x",
-      "Prefixes": [ "Kilo" ],
+      "Prefixes": [ "Kilo", "Mega" ],
       "Localization": [
         {
           "Culture": "en-US",
           "Abbreviations": [ "N/m²" ],
-          "AbbreviationsWithPrefixes": [ "kN/m²" ]
+          "AbbreviationsWithPrefixes": [ "kN/m²", "MN/m²" ]
```

Comment:
`AbbreviationsWithPrefixes` is only needed if prefixing `k` and `M` does not work, so for `en-US` culture it can be removed. I'll remove this in master after merging. Russian still needs to be kept.

Reply:
Fixed in https://github.com/angularsen/UnitsNet/pull/421